Base.rte/Effects/Pyro/Flame/Flame.lua

```diff
@@ -0,0 +1,52 @@
+function Create(self)
+	self.strength = 80;	-- Damage frequency factor
```

Comment:
If you've got a comment explaining that this is Damage frequency factor, and you use this variable in exactly 1 place, why not just call it `DamageFrequencyFactor` and nix the comment?